src/schemas/message.ts

```diff
@@ -20,6 +20,8 @@ export enum MessageType {
   Url         = 14,   // Url(5)
   Video       = 15,   // Video(4), Video(43)
   Post        = 16,   // Moment, Channel, Tweet, etc
+
+  System      = 18,   // System Message
```

Comment:
Why is this `18` instead of `17`?

Why is a blank line above the `System`?

In the future, we should fix the linting in a separate PR before we create other PRs.

Reply:
> Why is this `18` instead of `17`?
> 
> Why is a blank line above the `System`?
> 
> In the future, we should fix the linting in a separate PR before we create other PRs.

Because Channel was designed before System and it would be much easier for us if we keep 17 for Channel. (If we implemented Channle as Miniprogram etc)

Reply:
Yes, the lint pr has just been merged. I'll re do all the PRs to puppet this weekend.